sql/original_update_tests.md

```diff
@@ -0,0 +1,48 @@
+# Original Update Tests
+This file is a port of the original hardcoded update-tester tests to markdown, similar to how sql-doctester works.
```

Comment:
I'm not a fan of using markdown, as we aren't using any other features of markdown.  These files aren't intended to host documentation.  Seems confusing.